README.md

```diff
@@ -40,6 +40,15 @@ confirmation = agent.submit_financials(
 confirmation.success? # => true
 confirmation.errors # A hash of errors if errors occur
 
+# Submit financials w/ a bank account
+confirmation = agent.submit_financials(
+  bank_acct: '123456789',
```

Comment:
I'm not sure what our policy on abbreviations is, but I think that full words might be easier? No strong opinions, since `dob`, `prev` and `ssn` are all abbreviations too :/

Reply:
Yeah, agree. I'm thinking since this API isn't being used anywhere yet, we should take advantage of the opportunity to change it while we can.

Reply:
If so, my vote is for:

- `bank_routing_number`
- `bank_account_number-
- `bank_type`

Reply:
:+1:. Will do before merging